Makefile.PL

```diff
@@ -319,42 +310,26 @@ my %optional_features = (
             }
         }
     },
-    csp => {
-        description => 'Content-Security-Policy support',
-        prereqs     => {
-            runtime => {
-                requires => {
-                    'Type::Tiny' => 1,
-                    'MooX::StrictConstructor' => 0.008,
-                }
-            }
-        }
-    },
     elasticsearch => {
         description => 'Elasticsearch-powered searches',
-        prereqs => {
+        prereqs     => {
             runtime => {
-                recommends => {
-                    'Term::ProgressBar' => 0,
-                },
-                requires => {
-                    'Search::Elasticsearch' => 0,
-                    'IO::Async' => 0,
-                    'Role::Tiny' => 0,
-                },
-            }
+                recommends => { 'Term::ProgressBar'     => 0 },
+                requires   => { 'Search::Elasticsearch' => 0 },
```

Comment:
`Role::Tiny` was removed from the list (probably should be added to the `PREREQ_PM` list)

Reply:
Role::Tiny is effectively a part of Moo.

Reply:
i don't think it matters that it's highly unlikely that `Moo` will stop using `Role::Tiny`; that's missing the importance of listed out the modules that bmo uses directly.

bmo is using `Role::Tiny` directly.  `Moo` depends on `Role::Tiny`, however it isn't part of `Moo`, so `Role::Tiny` needs to be explicitly listed here.